Makefile

```
@@ -359,9 +360,9 @@ generate-client-mocks:
 #----------------------------------------------------------------------------------
 # build-ci and glooctl along with others are in the ci makefile
 
-
-
-
+ifeq ($(USE_SILENCE_REDIRECTS), true)
+STDERR_SILENCE_REDIRECT := 2> /dev/null
```

will this swallow important errors that we might want to see in case a build fails?

---

Yeah so this is the tradeoff here. My understanding is this will swallow the logs, if enabled, and we enable it in our regression suite. Our true build pipeline which also builds images is not swallowed.

This is why I was comfortable with this change. Should a build fail, I think we'd see the logs in build-bot. And even if we don't, we could easily enable the flag in CI and run again, or really the best is we should just run it locally as something is probably broken that we need to fix.